backend/src/main/resources/application-docker.yaml

```diff
@@ -0,0 +1,9 @@
+profile:
+  datasource:
+    url: postgresql://kirschbaum-postgres:5432/kirschbaum
+    user: admin
+    pass: 12345
+
+kirschbaum:
```

Comment:
I'm not sure why or if you need this

Reply:
Will be deleted, thanks for the effort! 👍